cmd/clef/main.go

```diff
@@ -351,6 +364,33 @@ func attestFile(ctx *cli.Context) error {
 	return nil
 }
 
+func listAccounts(c *cli.Context) error {
+	if err := initialize(c); err != nil {
+		return err
+	}
+	// listaccounts is meant for users using the CLI.
+	var (
+		ui                        = core.NewCommandlineUI()
+		pwStorage storage.Storage = &storage.NoStorage{}
+		ksLoc                     = c.String(keystoreFlag.Name)
+		lightKdf                  = c.Bool(utils.LightKDFFlag.Name)
+	)
+	am := core.StartClefAccountManager(ksLoc, true, lightKdf, "")
+	// Access external API and call List()
+	api := core.NewSignerAPI(am, 0, true, ui, nil, false, pwStorage)
+	accs, err := api.List(context.Background())
```

Comment:
This is a bit wonky :) 
We instantiate a cli-UI, but then we make a request via the _external_ api, about a listing. And since it's on the _external_ api, the user needs to approve it. 
It should be possible to get a listing via the internal api, originating _from the ui_, and thus _not_ requiring user-approval.

Reply:
Yep, there are 
```
  - `clef_listWallets`
  - `clef_listAccounts`
  - `clef_listWallets`
 ```
Or, if you want to invoke them directly, use `ListAccounts` 

```
	internalApi := core.NewUIServerAPI(api)
	accounts, err := internalApi.ListAccounts()
```

Reply:
Actually, we could add both `list-accounts` and `list-wallets` as CLI commands, corresponding to the two internal methods.

Reply:
ok i tried to fix this according to your comments and also added `clef list-wallets`. Hopefully this works.